Sources/SwiftSyntaxBuilder/gyb_generated/BuildableNodes.swift

```diff
@@ -2352,20 +2328,6 @@ public struct ClosureCaptureSignature: SyntaxBuildable, ExpressibleAsClosureCapt
     assert(rightSquare.text == "]")
   }
 
-  /// A convenience initializer that allows:
-  ///  - Initializing syntax collections using result builders
-  ///  - Initializing tokens without default text using strings
-  public init(
-    leftSquare: TokenSyntax = TokenSyntax.`leftSquareBracket`,
-    rightSquare: TokenSyntax = TokenSyntax.`rightSquareBracket`,
-    @ClosureCaptureItemListBuilder itemsBuilder: () -> ExpressibleAsClosureCaptureItemList? = { nil }
```

Comment:
🤷🏽‍♂️/✅   I don’t have strong opinions about this one. I’m fine with making it a result builder again because nobody is forced to use the result builder initializer.